conozco.py

```diff
@@ -22,49 +23,46 @@
 # Gabriel Eirea geirea@gmail.com
 # Alan Aguiar alanjas@hotmail.com
 
-import os.path
+import configparser
+import gettext
+import imp
```

Comment:
Use `importlib` as imp existed in python2 and no longer does.

Reply:
`python3 -c "import imp"` seems to work for me on 3.6.9.  Is there a specific version where it does not work?

Reply:
Using 3.7.3 importing it throws a DeprecationWarning.

Reply:
Thanks.  A fix is in

https://github.com/sugarlabs/iknowEditor/blob/master/conozco.py#L31-L32

https://github.com/sugarlabs/iknowEditor/blob/master/conozco.py#L251

Seems a lot of work to go through just to keep the file in a non-module directory, otherwise a plain import would be fine.  I wonder what was behind this design decision?

Reply:
Thanks, I will update accordingly